core/backend/package.json

```diff
@@ -43,6 +43,11 @@
     "@itwin/core-geometry": "workspace:^3.1.0-dev.27",
     "@itwin/ecschema-metadata": "workspace:^3.1.0-dev.27"
   },
+  "peerDependenciesMeta": {
+    "@itwin/ecschema-metadata": {
```

Comment:
we should probably also list core quantity because that is needed if you include ECSchema-metadata

Reply:
Not sure about this.  core-quantity should already be there once/if it's decided to install ecschema-metadata with the backend.  Meaning that if you install the backend you will have to install core-quantity.

Reply:
Yea, I agree with Chris, I think unless it's needed without ecschema-metadata imported then it's not necessary to list it.

Reply:
@ColinKerr you okay with us resolving this?